frontends/ast/simplify.cc

```diff
@@ -3609,10 +3582,11 @@ skip_dynamic_range_lvalue_expansion:;
 				AstNode *buf = children[0]->clone();
 				int mem_depth = 1;
 				int result, high = 0, low = 0, left = 0, right = 0, width = 1; // defaults for a simple wire
+				int expr_dimensions = 0, expr_unpacked_dimensions = 0;
 				AstNode *id_ast = NULL;
 
 				// Is this needed?
-				//while (buf->simplify(true, false, stage, width_hint, sign_hint, false)) { }
+				//while (buf->simplify(true, stage, width_hint, sign_hint)) { }
```

Comment:
No idea whether it can be needed in some situation, really - I only adapted the commented out code to earlier changes in Yosys.

In any case I've removed the comment now.